### DISCUSSION TITLE
Reflet PID SVA

### DISCUSSION THREAD

**Discussion by JYK-WIT** [witsa/synapps#564] [Q&A] [answered]
Bonjour, 
en utilisant le reflet PID SVA, je retrouve à l'affichage jusqu'à 8 décimales,
je n'arrive pas à l'arrondir à l'entier. y a-t-il une solution?
Merci
JYK

**Comment by WITNicolas** [comment]
Salut, je crois que tu peux faire un : parseInt(context.value).toFixed(0) via le OnReadTransform
(J'ai mal lu .... avec le Reflet je ne sais pas), je prendrais perso le .output de la sortie SVA

**Comment by JYK-WIT** [comment]
Merci Nico,  ça ne marche pas avec le reflet.
mais j'ai fait comme tu la suggéré ,J'ai mis le reflet (avec opacité à 0%) dessus un acteur text avec la valeur du.Output et le OnReadTransform que ma transmis.
Comme cela je peux toujours utilisé le reflet en forçage.
En attendant ça me vas
JYK

**Comment by manufitoussiwit** [comment, marked as answer]
J'ai une proposition pour utiliser uniquement le reflet : 

Dans l'évènement `onRequestDone`, ajouter ce script : 
```js
const reflect = context.data;
const precision = 2;
if (reflect) {
  reflect.output = parseFloat(reflect.output).toFixed(precision);
  reflect._setupStateText();
}
```
la variable `precision` est le nombre de chiffre après la virgule.

**Comment by JYK-WIT** [comment]
Merci Manu, mais je trouve pas l'événement OnRequestDone

**Comment by manufitoussiwit** [comment]
> Merci Manu, mais je trouve pas l'événement OnRequestDone

Sur l'acteur détails de reflet : 

![image](https://user-images.githubusercontent.com/35595723/203649357-1eafbb92-3e50-46f9-95e8-655558ab68c2.png)

**Comment by JYK-WIT** [comment]
Ok j’avais mis le reflet dans un composite , donc je n’avais pas accès à l’événement
